doc/modules/classes.rst

```diff
@@ -1317,6 +1317,7 @@ Visualization
    naive_bayes.ComplementNB
    naive_bayes.GaussianNB
    naive_bayes.MultinomialNB
+   naive_bayes.ColumnwiseNB
```

Comment:
I glanced over the `naive_bayes` estimator from R [here](https://www.rdocumentation.org/packages/naivebayes/versions/0.9.7/topics/naive_bayes). I don't know if automatic selection of sub-estimators based on data type is a good idea. The R's `naive_bayes` has already run into the problem of specifying which distribution to use. They had to introduce `usepoisson` argument to allow the user to choose the distribution, yet this choice must be made for *all* features of integer type together. Any additional flexibility will likely spawn additional arguments with confusing names... I think our approach is better.

And since our approach is different, there is less reason to mimic the name. I think an average sklearn user will find the reference to `ColumnTranformer` helpful in understanding how `ColumnwiseNB` works. Users who switch from R will have to first learn `ColumnTranformer` anyway.